am-lib/src/main/java/uk/gov/hmcts/reform/amlib/exceptions/UnsupportedPermissionsException.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.reform.amlib.exceptions;
+
+public class UnsupportedPermissionsException extends AccessManagementException {
```

Comment:
I think it makes more sense to make based of RuntimeException. Permissions will be validated and chances of getting UnsupportedPermissionsException should be extremely low. Since that IMO there is no point declaring that exception on every single method that could potentially throw it.